index.qmd

```diff
@@ -589,6 +635,15 @@ TimberV$inventory %>%
   ggplot(aes(DeathCause, AGB)) + geom_col() + xlab("")
 ```
 
+<!-- Actually, if you do it like that you just get the cause of death but not really the volume of damage.
+We indeed considered that the unused part of harvested trees are also damage.
+So either we keep the graph like that and change the text, or we change the graph to get the actual quantity of damage.
+I'm in favour of the second option.
+
+I made a function for that for ManagForRes, I will adapt it for the presentation to practitionners, we can use it here if we decide to go for that.
+Note for me; it's in Misc/2211_check_deathtree_fate.Rmd
+-->
```

Comment:
Ok, let's rediscuss it. But the idea was to have a very short code.

Reply:
@sylvainschmitt maybe we don't have to do exactly the same think for the pratice and for the presentation to students.
So probably you should keep it the way you did it for the students, but saying that this is the fate of the trees, not the biomass of damage.
And ofr the practitionners, I'll do the volume of actual damage, without having to explain the code.
OK ?

Reply:
it actually wouldn't be a long code. I cannot remember why it was long, I think Vincyane added stuff to the output after I did the work for ManagForRes. See my code around line 128

Reply:
Line 129 of which code / document? But if this is short I agree.

Reply:
Ok this is fixed now, I'll push it online soon.